### DISCUSSION TITLE
Budibase and API DIGEST Authentication in Budibase

### DISCUSSION THREAD

**Discussion by dayeehook** [Budibase/budibase#5192] [Help]
Hi all,

I'm just getting started with Budibase to perform a proof of concept for our company but I'm getting stuck on one portion.  I need to ensure that an API pull on some data can occur from a vendor of ours (and from there, we can perform additional queries).

However, the issue that i'm running into is how to perform a DIGEST AUTH in Budibase.  I see the "none" and "basic auth" options but no Digest.  How did users of the Budibase system get that to work when needed?  Would appreciate any direction on this.

Now i'm also new to the world of APIs but i've got the same API working in VBA and I can cURL the request without issue elsewhere.  Just can't figure it out here in Budibase.

Thanks in advance for any feedback.  If you have questions for me, feel free to ask.

**Comment by melohagan** [comment]
Hi @dayeehook 

There is not direct support for digest auth in budibase, however you could try manually adding the client auth headers.
